util/crossgcc/buildgcc

```diff
@@ -52,7 +52,8 @@ MPFR_ARCHIVE="https://ftpmirror.gnu.org/mpfr/mpfr-${MPFR_VERSION}.tar.xz"
 MPC_ARCHIVE="https://ftpmirror.gnu.org/mpc/mpc-${MPC_VERSION}.tar.gz"
 GCC_ARCHIVE="https://ftpmirror.gnu.org/gcc/gcc-${GCC_VERSION}/gcc-${GCC_VERSION}.tar.xz"
 BINUTILS_ARCHIVE="https://ftpmirror.gnu.org/binutils/binutils-${BINUTILS_VERSION}.tar.xz"
-IASL_ARCHIVE="https://acpica.org/sites/acpica/files/acpica-unix2-${IASL_VERSION}.tar.gz"
+# acpica.org links rotted, use Intel mirror for 20220331
+IASL_ARCHIVE="https://downloadmirror.intel.com/774879/acpica-unix2-${IASL_VERSION}.tar.gz"
```

Comment:
Why don't we use the same lin kas here? https://github.com/Dasharo/coreboot/pull/366

IASL_VERSION is 20220331 here as well @tlaurion

Reply:
@miczyg1 as you see fit. Same will apply to kgpe-d6 dasharo fork and all other forks. This was just a guideline to be followed, meanwhile staging patches missing newer commit under Heads lies into forks staging dirs.